apps/settings/style/screen_lock.css

```diff
@@ -168,12 +168,12 @@ span.cb-passcode-digit[data-dot]::before {
   content: '';
   display: block;
   position: absolute;
-  width: 1.5rem;
-  height: 1.5rem;
+  width: 0.7rem;
+  height: 0.7rem;
   background-color: #3e3b39;
   border-radius: 0.75rem;
-  top: 50%;
-  left: 50%;
+  top: 64%;
```

Comment:
hmm, can we use rem values here instead of % ? For me 64% and 56% on a next line looks suspicious and it feels like a 'hack'. If you wanted to get a precise px values here, but not in px, than let's consider that 10px == 1rem.